cmd/scaffold.go

```diff
@@ -135,6 +135,7 @@ type FlowNodeBuilder struct {
 	postInitFns       []func(*FlowNodeBuilder)
 	stakingKey        crypto.PrivateKey
 	networkKey        crypto.PrivateKey
+	unstaked          bool
```

Comment:
this flag is used to determine if we are running a staked or an unstaked access node.
I had to be here in scaffold.go and not in the `cmd/access/main.go` file because the `initNodeInfo` happens here which initializes the node info (node id, keys, address etc.).